source/includes/_invoice.md

```diff
@@ -2658,6 +2658,357 @@ Note that for `SUBSCRIPTION_ACTION`, there are 2 dates to take into account:
 
 If successful, returns a status code of 200 and an invoice resource object.
 
+## Invoice Groups
+
+An invoice group is a group of invoices. An invoice group plugin can be used to split an invoice into multiple invoices. Such split invoices belong to a single invoice group. 
+
+Note that Kill Bill does not provide any reference implementation for an invoice group plugin. So anyone wishing to use this feature would need to create an invoice plugin and implement the [InvoicePluginApi#getInvoiceGrouping](https://github.com/killbill/killbill-plugin-api/blob/20e10d463b67b0434e1b9f4dd62ed34ebe84bd9b/invoice/src/main/java/org/killbill/billing/invoice/plugin/api/InvoicePluginApi.java#L41) method. We provide a [demo plugin](https://github.com/killbill/killbill-invgrp-demo) that demonstrates how an invoice group plugin can be implemented. When such a plugin is configured, an [invoice group run](https://killbill.github.io/slate/#invoice-trigger-an-invoice-group-run) would result in the generation of multiple invoices.
```

Comment:
@reshmabidikar I had a discussion with @pierre , and we thought we would rename the [killbill-invgrp-demo](https://github.com/killbill/killbill-invgrp-demo) to something like `killbill-invgrp` - after all this is a fully functional plugin. In which, we could then say that such a plugin can be used as a reference implementation.

I don't suggest any changes for now, but if you could track it, then we could update plugin name and doc together.

Reply:
Acknowledged, I've filed https://github.com/killbill/technical-support/issues/89 to track this.